admin/app/components/solidus_admin/ui/modal/component.html.erb

```diff
@@ -0,0 +1,33 @@
+<div data-controller=<%= stimulus_id %> id=<%= @id %> class="relative z-10 hidden" aria-labelledby="modal-title" role="dialog" aria-modal="true">
```

Comment:
Thoughts on using the `<dialog>` element directly instead of using `role="dialog"`?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog